SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -123,9 +123,17 @@ void FLocalDeploymentManager::StartUpWorkerConfigDirectoryWatcher()
 
 void FLocalDeploymentManager::OnWorkerConfigDirectoryChanged(const TArray<FFileChangeData>& FileChanges)
 {
-	UE_LOG(LogSpatialDeploymentManager, Log,
-		   TEXT("Worker config files updated. Regenerating worker descriptors ('spatial worker build build-config')."));
-	WorkerBuildConfigAsync();
+	for (auto& FileChange : FileChanges)
```

Comment:
nit 
```suggestion
	for (const auto& FileChange : FileChanges)
```
Interesting a TArray returns non-const iterators